src/roadmap/release-schedule.md

```diff
@@ -1,95 +1,82 @@
 # Release Schedule
 
-Some of the greatest assets of the Deis project are velocity and agility.
-Deis changed rapidly and with relative ease during initial development.
+Some of the greatest assets of the Deis team are velocity and agility. Workflow changed rapidly and
+with relative ease during initial development.
 
-Deis now harnesses those strengths into powering a regular, public release
-cadence. From v1.0.0 onward, the Deis project will release a minor version each
-month, with patch versions as needed. The project will use GitHub milestones to
-communicate the content and timing of major and minor releases.
+Deis now harnesses those strengths into powering a regular, public release cadence. From v2.0.0
+onward, the Deis team will release a minor version every 2 weeks, with patch versions as needed.
+The project will use GitHub milestones to communicate the content and timing of major and minor
+releases.
 
-Deis releases are not feature-based, in that dates are not linked to specific
-features. If a feature is merged before the release date, it is included in the
-next minor or major release.
+Deis releases are not feature-based, in that dates are not linked to specific features. If a
+feature is merged before the release date, it is included in the next minor or major release.
 
-The master `git` branch of Deis always works. Only changes considered ready to
-be released publicly are merged, and releases are made from master.
+The master `git` branch of Deis should always work. Only changes considered ready to be released
+publicly are merged, and non-patch releases are made from master.
```

Comment:
Even that might not be clear enough. I believe we sometimes _will_ cut a patch release from master. This would happen when master doesn't contain any new features from the previous release.